src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -897,6 +912,9 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
         {templateID &&
           HAS_REACTIONS_TEMPLATES.includes(templateID) &&
           renderShowReactions()}
+        {templateID &&
+          HAS_REACTIONS_TEMPLATES.includes(templateID) &&
+          renderShowShoutouts()}
```

Comment:
`HAS_REACTIONS_TEMPLATES` Should be updated to include all locations that have shoutouts, and then we should be adding the check to each of those templates to ensure this toggle is 'connected' to them.